### PR TITLE
Update sl-jdbc.md

### PR DESCRIPTION
**Pull request by rpourzand**
I realized the examples in the jdbc api query table are missing single quotes! I added them

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 1]
```diff
@@ -135,8 +135,8 @@ To query metric values, here are the following parameters that are available:
...
 | Parameter | Description  | Example    | Type |
 | --------- | -----------| ------------ | -------------------- |
-| `metrics`   | The metric name as defined in your dbt metric configuration   | `metrics=[revenue]` | Required    |
-| `group_by`  | Dimension names or entities to group by. We require a reference to the entity of the dimension (other than for the primary time dimension), which is pre-appended to the front of the dimension name with a double underscore. | `group_by=[user__country, metric_time]`     | Optional   |
+| `metrics`   | The metric name as defined in your dbt metric configuration   | `metrics=['revenue']` | Required    |
+| `group_by`  | Dimension names or entities to group by. We require a reference to the entity of the dimension (other than for the primary time dimension), which is pre-appended to the front of the dimension name with a double underscore. | `group_by=['user__country', 'metric_time']`     | Optional   |
 | `grain`   | A parameter specific to any time dimension and changes the grain of the data from the default for the metric. | ```group_by=[`Dimension('metric_time').``` <br/> ```grain('week\|day\|month\|quarter\|year')]``` | Optional     |
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: @rpourzand should "Dimension" not have the ` as a prefix? So should it be just Dimension without any quotes?

```suggestion
| `grain`   | A parameter specific to any time dimension and changes the grain of the data from the default for the metric. | `group_by=[Dimension('metric_time')` <br/> `grain('week\|day\|month\|quarter\|year')]` | Optional     |
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 2]
```diff
@@ -135,8 +135,8 @@ To query metric values, here are the following parameters that are available:
...
 | `where`     | A where clause that allows you to filter on dimensions and entities using parameters  - comes with `TimeDimension`, `Dimension`, and `Entity` objects. Granularity is required with `TimeDimension`  | `"{{ where="Dimension('customer__country') = 'US')"`   | Optional   |
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: same as above, should Dimension have quotes or no quotes?

```suggestion
| `where`     | A where clause that allows you to filter on dimensions and entities using parameters  - comes with `TimeDimension`, `Dimension`, and `Entity` objects. Granularity is required with `TimeDimension`  | `"{{ where=Dimension('customer__country') = 'US')"`   | Optional   |
```

**Comment by mirnawong1** [comment]
hey @rpourzand thanks for opening this up! i left some questions for other fields as they may not need apostrophes -- wanted to double check with you!

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md (hunk 1 above)
Comment (rpourzand) [reply website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: Yeah, Dimension doesn't need quotes just the objects within the parens!

**Review by mirnawong1 — APPROVED** [review]
hey @rpourzand , looks good! approving to merge ✨ ! thanks for opening this up!

**Pull request merged**